README.md

```diff
@@ -0,0 +1,43 @@
+# Plane Mail Management Initial Documentation
+
+### Onboarding Instructions for Testing on Windows
+
+1. Go to "Turn Windows features on or off" in the control panel
+
+2. Enable "Windows Subsystem for Linux" checkbox. You may have to restart your computer for changes to take effect
+
+3. Install Ubuntu
+
+4. Choose a location and create a folder to store the content for Plane Mail. You can use cd ~ to create the folder in a separate file system from your local drive.
+
+5. Install python 3 and pip if you have not already done so. You can run "sudo apt update" and "sudo apt install python3-pip"
+
+6. Create a virtual environment by running: "python3 -m venv {name}"
+
+7. Activate environment by running: "source {name}/bin/activate"
+
+8. Install requirement files by running: "python3 -m pip install -r requirements.txt"
+
+9. Git clone the repository in the environment you have created (git clone https://github.com/ScottyLabs/plane.git). Move into the new branch you are editing on.
+
+10. cd into the "plane" folder and transfer the file secrets.py into the "plane" folder
+
+11. Run plane.py for testing
+
+### Onboarding Instructions for Testing on Mac
+
+1. Create a new folder to store the contents of the Plane Mail Management repository
+
+2. In command prompt, cd into the folder you have just created
+
+3. Create a virtual environment by running: "python3 -m venv {name}"
+
+4. Activate environment by running: "source {name}/bin/activate"
+
+5. Install requirement files by running: "python3 -m pip install -r requirements.txt"
+
+6. Git clone the repository in the environment you have created (git clone https://github.com/ScottyLabs/plane.git). Move into the new branch you are editing on
+
+7. cd into the "plane" folder and transfer the file secrets.py into the "plane" folder
```

Comment:
Maybe include instructions on how to transfer the file? I just did touch secrets.py in the directory and then copy-pasted the contents in there since I forgot how to scp files in.